7_fetalSC_vs_LGR5+_and_LGR5-/0_Haber_2017_create_anndata_file.ipynb

```diff
@@ -0,0 +1,2898 @@
+{
```

Comment:
<p>1) This definitely needs a header. I had to take a minute and go trhought the thing to know hta you were building an anndata object.</p><p>2)  I have never seen this before. Why we can't use the <code>scanpy</code> read dataframe function here? <code>sc.read_csv</code> ?</p><p>3) Do we need to do the TPM matrix? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>